test/test_kfac.py

```diff
@@ -426,3 +429,88 @@ def test_bug_device_change_invalidates_parameter_mapping():
     kfac_x_cpu = kfac @ x
 
     report_nonclose(kfac_x_gpu, kfac_x_cpu)
+
+
+@mark.parametrize("dev", DEVICES, ids=DEVICES_IDS)
+def test_torch_matmat(dev: device):
```

Comment:
Can we use `cases` just like for the inverse KFAC tests?

Reply:
Leaving this open for discussion. I believe it would make the code shorter and also allow testing different `dtypes` in the long run (which atm we are not).